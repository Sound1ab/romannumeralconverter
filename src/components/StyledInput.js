import styled from 'styled-components';

export const StyledInput = styled.input`
  padding: 16px 16px 16px 60px;
  margin: 48px auto;
  border: none;
  background: rgba(0, 0, 0, 0.2);
  box-shadow: inset 0 -2px 1px rgba(0, 0, 0, 0.03);
  color: white;
  display: block;
  outline: none;
`;