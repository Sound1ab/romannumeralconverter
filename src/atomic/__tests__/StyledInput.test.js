import React from 'react';
import {StyledInput} from "../StyledInput";
import renderer from 'react-test-renderer';
import 'jest-styled-components';

describe('StyledError', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<StyledInput>Error Message</StyledInput>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
