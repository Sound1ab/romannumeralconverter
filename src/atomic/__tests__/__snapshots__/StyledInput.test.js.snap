// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledError renders correctly 1`] = `
.c0 {
  padding: 16px 16px 16px 60px;
  margin: 48px auto;
  border: none;
  background: rgba(0,0,0,0.2);
  box-shadow: inset 0 -2px 1px rgba(0,0,0,0.03);
  color: white;
  display: block;
  outline: none;
}

<input
  className="c0"
>
  Error Message
</input>
`;
